react/package.json

```diff
@@ -19,7 +19,7 @@
     "web-vitals": "~1.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-app-rewired start",
```

Comment:
This breaks HMR, see https://github.com/sentry-demos/empower/pull/381

Need to rewire jsx-runtime but also want HMR... 🤔

Reply:
HMR was already broken, but I reverted this change anyway...